setup.py

```diff
@@ -22,12 +22,14 @@
     python_requires=">=3.8",
     install_requires=[
         "requests",
+        "attrs",
         "openeo>=0.9.3.a2.dev",
         "openeo_driver>=0.22.0.dev",
         "flask~=2.0",
         "gunicorn~=20.0",
         "python-json-logger>=2.0.0",
         "kazoo~=2.8.0",
+        "deepdiff~=5.8.1"
```

Comment:
TODO: can this `~=5.8.1"` constrained be relaxed to a `>=`  ?  or at least document why it is pinned down that much

Reply:
(same for that kazoo dep actually)